modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -77,7 +81,8 @@ To enable/disable APIs, you have to:
      (e.g. <tt>-DWITH_MSMF=ON -DWITH_VFW=ON ... </tt>) or checking related switch in cmake-gui
   2. rebuild OpenCV itself
 */
-enum { CAP_ANY          = 0,            //!< Auto detect
+enum VideoCaptureApiID {
```

Comment:
I suggest removing `ID` suffix from all identifiers and use plural forms: `VideoCaptureAPIs`, `VideoCaptureProperties`, `VideoCaptureModes`, etc.